mentorship/slack_coc.md

```diff
@@ -0,0 +1,70 @@
+- **Community organizers** wanting to connect their communities with this initiative.
+
+By "R core" we refer to the [base packages](https://cran.r-project.org/doc/FAQ/R-FAQ.html#Which-add_002don-packages-exist-for-R_003f), which R users outside of the [R Core Team](https://www.r-project.org/contributors.html) may contribute to through discussion of bugs/new features, [reviewing bug reports](https://developer.r-project.org/Blog/public/2019/10/09/r-can-use-your-help-reviewing-bug-reports/index.html) and proposing patches. On the Slack we will discuss coding issues, the wider development process and outreach initiatives. 
+
+## Posting Guide
+
+To create a welcoming and supportive workspace, we ask members to:
+- Be welcoming, friendly and patient.
+- Avoid sarcasm or belittling language.
+- Respect people's technology choices.
+- Provide direct links vs "RTFM"/"Just Google it" responses.
+- Use gender-neutral language.
+
+The [administrators](https://r-devel.slack.com/account/workspace-settings#admins) will remind people of the posting guide if necessary and reserve the right to deactivate the membership of anyone persistently acting against the ethos of the workspace.
+
+**Messages on public channels of the Slack may be put in a public archive or shared outside the Slack**. Use DMs/private channels for private messages, which should not be shared without consent.
+
+By joining the Slack, members also agree to the following general code of conduct. The administrators will enforce this code of conduct and expect cooperation in ensuring a safe environment for all. 
+
+## Code of Conduct
+
+The [administrators](https://r-devel.slack.com/account/workspace-settings#admins) of R-dev Slack are dedicated to providing a harassment-free workspace for everyone regardless of age, gender, sexual orientation, disability, physical appearance, race, or religion (or lack thereof). 
+
+### Expected Behaviour
+
+All members agree to: 
+
+- Be considerate in language and actions, and respect the boundaries of fellow members.
+- Refrain from demeaning, discriminatory, or harassing behaviour and language.  Please refer to [Unacceptable Behaviour](#unacceptable-behaviour) for more details. 
+- Alert one of the [administrators](https://r-devel.slack.com/account/workspace-settings#admins) if you observe violations of this code of conduct, even if they seem inconsequential.
+
+### Unacceptable Behaviour
+
+Behaviour that is unacceptable includes, but is not limited to:
+
+ - Stalking, e.g. persistent unwanted DMs, logging online activity for harassment purposes.
+ - Deliberate intimidation.
+ - Sustained or willful disruption of discussion.
+ - Use of sexual or discriminatory imagery, comments, or jokes.
+ - Offensive comments related to age, gender, sexual orientation, disability, race or religion.
+ - Inappropriate simulated physical contact, e.g. textual descriptions like "hug" or "backrub", without consent.
```

Comment:
Is this meant to apply to emoji use within Slack as well (e.g. 🤗)? If so, it could be helpful to provide a concrete example.

Reply:
Yes, I don't think any of the default Slack emoji could violate the CoC, but custom emoji are allowed, so I added "or custom emoji representing such contact"

Reply:
Ah, ok. Thanks for clarifying! I've never made a custom emoji in Slack so that possibility didn't even cross my mind